index.html

```diff
@@ -1,124 +1,193 @@
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
 
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
 
-  <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-149400351-4"></script>
-  <script>
-    window.dataLayer = window.dataLayer || [];
+      gtag("config", "UA-149400351-4");
+    </script>
+  </head>
 
-    function gtag() {
-      dataLayer.push(arguments);
-    }
-    gtag('js', new Date());
-
-    gtag('config', 'UA-149400351-4');
-  </script>
-</head>
-
-<body>
-  <div class="shade"></div>
-  <header>
-    <div class="navbar fixed-top">
-      <button type="button" class="sidebar-toggler d-none d-block d-md-none">
-        <i class="fas fa-bars"></i>
-      </button>
-      <div class="logo">
-        <img src="assets/img/sbuttons-nav.png" class="img-fluid" />
-      </div>
-      <div class="buttons">
-        <a class="toggle-theme" href="#toggleTheme" title="Toggle light/dark mode">
-          <i class="fas fa-moon"></i>
-        </a>
-        <a href="#" id="downloadGithubRawHeader" class="download-link">
-          <i class="fas fa-download"></i>
-        </a>
-        <a href="https://github.com/sButtons/sbuttons" target="_blank" class="icon-link">
-          <i class="fab fa-github"></i>
-        </a>
-      </div>
-    </div>
-    <div class="content">
-      <div class="logo">
-        <img src="assets/img/sbuttons.png" class="img-fluid" />
+  <body>
+    <div class="shade"></div>
+    <header>
+      <div class="navbar fixed-top">
+        <button type="button" class="sidebar-toggler d-none d-block d-md-none">
+          <i class="fas fa-bars"></i>
+        </button>
+        <div class="logo">
+          <img src="assets/img/sbuttons-nav.png" class="img-fluid" />
+        </div>
+        <div class="buttons">
+          <a
+            class="toggle-theme"
+            href="#toggleTheme"
+            title="Toggle light/dark mode"
+          >
+            <i class="fas fa-moon"></i>
+          </a>
+          <a href="#" id="downloadGithubRawHeader" class="download-link">
+            <i class="fas fa-download"></i>
+          </a>
+          <a
+            href="https://github.com/sButtons/sbuttons"
+            target="_blank"
+            class="icon-link"
+          >
+            <i class="fab fa-github"></i>
+          </a>
+        </div>
       </div>
-      <div class="subtitle">
-        Simple buttons you can use easily for your next project.
+      <div class="content">
+        <div class="logo">
+          <img src="assets/img/sbuttons.png" class="img-fluid" />
+        </div>
+        <div class="subtitle">
+          Simple buttons you can use easily for your next project.
+        </div>
+        <a
+          href="#"
+          id="downloadGithubRawButton"
+          class="banner-link sbtn transparent-btn white-btn"
```

Comment:
Please also remove banner-link as it is not necessary anymore

Reply:
Sure Will do in few minutes